TrackPerf/TruthHists.hxx

```diff
@@ -14,10 +16,12 @@ class TruthHists {
   TruthHists& operator=(const TruthHists&) = delete;
 
   //! Initialize empty histograms
-  TruthHists();
+  TruthHists(bool effi);
 
   // Fill histograms with a single track
-  void fill(const EVENT::MCParticle* track);
+  void fill(const EVENT::MCParticle* particle);
+  void effi(const EVENT::MCParticle* particle, bool passed);
+  void deltaR(const EVENT::MCParticle* particle, const EVENT::Track* track);
```

Comment:
Histograms correlating `MCParticle` and `Track` properties should go into `ResoHists`.